pandas/core/series.py

```diff
@@ -2101,8 +2101,16 @@ def map(self, arg, na_action=None):
         na_action : {None, 'ignore'}
             If 'ignore', propagate NA values
```

Comment:
Can you maybe add here "without passing them to the mapper" or something? (not sure about 'mapper', but not directly thinking of a better word).